model/src/main/proto/arguments.proto

```diff
@@ -29,3 +29,28 @@ enum HighlightItem {
   // Indicates that the Developer Options item needs to be highlighted.
   DEVELOPER_OPTIONS_ITEM = 4;
 }
+
+// Arguments needed to open the policy page.
+message PoliciesActivityArguments {
+  // The specific policy page that should be displayed.
+  PolicyPage policy_page = 1;
+}
+
+// Arguments needed to open the policy page.
+message PoliciesFragmentArguments {
+  // The specific policy page that should be displayed.
+  PolicyPage policy_page = 1;
+}
+
```

Comment:
Do we actually need these two messages? if the policy page enum is the only thing thats required, cant we just use that directly in the activity/fragment arguments? Or are we going to have more arguments in each of these places?

Reply:
We need this message to pass it between the activities and fragments. We can avoid using two and make it one generic message for both activity/fragments, that would remove redundancy.

Reply:
Marking this as resolved since it appears to be, and Vinita approved the PR. Trying to manage the comment threads since there are several that still need to resolved.

@vinitamurthi please let me know if you disagree with this resolution.